java/ql/test/library-tests/dataflow/collections/Test.java

```diff
@@ -84,8 +84,30 @@ public void run4() {
     p.put("key", tainted);
     sink(p.getProperty("key")); // Flow
     sink(p.getProperty("key", "defaultValue")); // Flow
-    
+
+    Properties clean = new Properties();
+    sink(clean.getProperty("key", tainted)); // Flow
```

Comment:
This is accidentally duplicated on lines 98-99

Reply:
Deduplicated :)